x-pack/test/security_solution_endpoint_api_int/apis/resolver/tree.ts

```diff
@@ -3,232 +3,28 @@
  * or more contributor license agreements. Licensed under the Elastic License;
  * you may not use this file except in compliance with the Elastic License.
  */
-import _ from 'lodash';
 import expect from '@kbn/expect';
 import {
-  ResolverChildNode,
-  ResolverLifecycleNode,
   ResolverAncestry,
-  ResolverEvent,
-  ResolverRelatedEvents,
   ResolverChildren,
   ResolverTree,
   LegacyEndpointEvent,
-  ResolverNodeStats,
-  ResolverRelatedAlerts,
 } from '../../../../plugins/security_solution/common/endpoint/types';
-import {
-  parentEntityId,
-  eventId,
-} from '../../../../plugins/security_solution/common/endpoint/models/event';
+import { parentEntityId } from '../../../../plugins/security_solution/common/endpoint/models/event';
```

Comment:
:grey_question: Should we do the safe version of this?

Reply:
Unfortunately we're going to have to stick with the unsafe version until we change most of the backend implementation over to use the safe events I think. When I try to use it I get this:

![image](https://user-images.githubusercontent.com/56361221/91180954-530cb700-e6b6-11ea-9c42-1733337e0760.png)